### PR TITLE
add functionality to startup profile page

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #28

### CODE REVIEW

File: src/mock_data.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  data: [{
+    founderName: 'mavis',
+    startupName: 'something',
+    phoneNumber: '0524645',
```
Comment (Karyum) [inline src/mock_data.js]: there is no phone number in the schema so i dont think you need it

File: src/routes/index.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
 const router = require('express').Router();
 
 const searchHome = require('./search_home.js');
+const startupProfile = require('./startup_profile.js');
 
 router.get('/', searchHome);
+router.get('/startupProfile', startupProfile);
```
Comment (Karyum) [inline src/routes/index.js]: so what happens when someone goes to this endpoint?

File: src/views/startup_profile.hbs [hunk 3]
```diff
@@ -1,29 +1,16 @@
...
-    <h2 class="">{{this.startupName}}</h2>
+    <h2 class="">{{startup.startupName}}</h2>
   </div>
   <div class="">
-    <span class="">{{this.founderName}}</span>
-    <span class="">{{this.phoneNumber}}</span>
-    <span class="">{{this.email}}</h2>
-    <span class="">{{this.startupField}}</span>
+    <span class="">{{startup.founderName}}</span>
+    <span class="">{{startup.phoneNumber}}</span>
```
Comment (Karyum) [inline src/views/startup_profile.hbs]: no phone number in the schema

File: src/routes/index.js (hunk 2 above)
Comment (Suuuuha) [inline src/routes/index.js]: changed it .. he will get an error

File: src/mock_data.js (hunk 1 above)
Comment (Suuuuha) [inline src/mock_data.js]: removed

File: database/build_mock.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+mongoose.connect(process.env.DB_URL);
+
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', () => {
+  // We are connected
+  const startup = new mockCollection(mockData.data[0])
+  startup.save((error, res) => {
+    if(error) console.log(error);
+    console.log('startup added');
```
Comment (esraajb) [inline database/build_mock.js]: let's add an else in here so the code will be clearer

File: src/routes/startup_profile.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+const { mockCollection } = require('../../database/schema.js');
+
+module.exports = (req, res) => {
+  const id = req.params.id;
+  mockCollection.findOne({_id:id}, (error, startup)=> {
+    if(error) res.render(error);
```
Comment (esraajb) [inline src/routes/startup_profile.js]: So this is the case where someone clicks on a startup logo/name on the results page and for some reason, we won't find it in our database. 
in this case, we should handle the error in a better way. rendering 'error' won't be the best user experience.

**Comment by Suuuuha** [comment]
@esraajb made changes !

**Comment by Suuuuha** [comment]
@esraajb made changes !

**Pull request merged**